src/main/scala/com/amazon/deequ/suggestions/rules/NonNegativeNumbersRule.scala

```diff
@@ -26,8 +26,7 @@ case class NonNegativeNumbersRule() extends ConstraintRule[ColumnProfile] {
 
   override def shouldBeApplied(profile: ColumnProfile, numRecords: Long): Boolean = {
     profile match {
-      case numericProfile: NumericColumnProfile
-        if numericProfile.minimum.isDefined && numericProfile.minimum.get == 0.0 => true
+      case numericProfile: NumericColumnProfile => numericProfile.minimum.exists(_ >= 0.0)
```

Comment:
Is this merely a simplification of the code or a change in functionality?

Reply:
This changes the behavior such that non negative constraints should be applied if the minimum of a column is >= 0.